cmd/query/app/server.go

```diff
@@ -111,11 +111,15 @@ func createGRPCServer(querySvc *querysvc.QueryService, options *QueryOptions, lo
 	server := grpc.NewServer(grpcOpts...)
 
 	handler := NewGRPCHandler(querySvc, logger, tracer)
+
+	// TODO: Register MetricsQueryService
```

Comment:
Will be addressed in a follow-up PR implementing MetricsQuery support for the gRPC handler.